scripts/README.md

```diff
@@ -2,20 +2,20 @@
 
 ## update_library_version.sh
 
-This script updates the version of a specified library in Pipfile and requirements-elyra.txt files, only if the new version is higher. It can optionally run `pipenv lock` after updating the version.
+This script updates the version of one or more libraries in Pipfile and requirements-elyra.txt files, only if the new version is higher. It can optionally run `pipenv lock` after updating the version.
 
 ### Examples
 
-Update the `numpy` library to version `2.0.1` in all files under `./myproject`, and run `pipenv lock`:
+Update the `numpy` library to version `2.0.1` and the `pandas` library to version `2.2.2` in all files under the current folder (`.`), and run `pipenv lock`:
 
 ```sh
-./update_library_version.sh ./myproject numpy 2.0.1 '' '' true
+./scripts/update_library_version.sh . '[{"name":"numpy","version":"2.0.1"},{"name":"pandas","version":"2.2.2"}]' '' '' true
```

Comment:
preparing correctly formatted json like this is quite a lot of work; just saying

Reply:
/lgtm